Libraries/Animated/components/AnimatedFlatList.js

```diff
@@ -10,8 +10,8 @@
 
 import type {AnimatedComponentType} from '../createAnimatedComponent';
 
-import FlatList from '../../Lists/FlatList';
 import createAnimatedComponent from '../createAnimatedComponent';
+import {FlatList} from '@react-native/flat-lists';
```

Comment:
What we did with virtualized list is kept all the existing files and imported the exports from the new package. That way none of the internal paths in RN change, and imports outside of FlatList won't be to be rewritten.

Reply:
Sure. Will rebase and try to keep it that way...

ps: Thanks for reply :)

Reply:
If it helps you can refer to the virtualized-lists package PR: https://github.com/facebook/react-native/pull/35406

Reply:
Rebased on the main and updated.